src/transfer.jl

```diff
@@ -168,6 +76,70 @@ function transfer_blochwave_kpt(ψk_in, basis_in::PlaneWaveBasis{T}, kpt_in::Kpo
     ψk_out
 end
 
+"""
+Transfer an array `ψk_in` expanded on `kpt_in`, and produce ``ψ(r) e^{i ΔG·r}`` expanded on
+`kpt_out`. It is mostly useful for phonons.
+Beware: `ψk_out` can lose information if the shift `ΔG` is large or if the `G_vectors`
+differ between `k`-points.
+"""
+function transfer_blochwave_kpt(ψk_in, basis::PlaneWaveBasis, kpt_in, kpt_out, ΔG)
+    ψk_out = zeros(eltype(ψk_in), length(G_vectors(basis, kpt_out)), size(ψk_in, 2))
+    for (iG, G) in enumerate(G_vectors(basis, kpt_in))
+        # e^i(kpt_in + G)r = e^i(kpt_out + G')r, where
+        # kpt_out + G' = kpt_in + G = kpt_out + ΔG + G, and
+        # G' = G + ΔG
+        idx_Gp_in_kpoint = index_G_vectors(basis, kpt_out, G - ΔG)
+        if !isnothing(idx_Gp_in_kpoint)
+            ψk_out[idx_Gp_in_kpoint, :] = ψk_in[iG, :]
+        end
+    end
+    ψk_out
+end
+
+"""
+Real-space version of the previous function.
```

Comment:
Is it generally a good idea to do this? I mean isn't this always slower than `transfer_blochwave_kpt` and just useful for testing?